### PR TITLE
Make hyperkit driver more robust: detect crashing, misinstallation, other process names

### PR DESCRIPTION
**Pull request by tstromberg**
Things I ran into today when I had an older hyperkit driver installed and tried to update it, badly.

Also adds logging for DHCP lease discovery.

### CODE REVIEW

**Comment by tstromberg** [comment]
@minikube-bot OK to test

File: pkg/drivers/hyperkit/network.go [hunk 1]
```diff
@@ -120,12 +120,11 @@ func trimMacAddress(rawUUID string) string {
 }
 
 func GetNetAddr() (net.IP, error) {
-	_, err := os.Stat(CONFIG_PLIST + ".plist")
-	if err != nil {
-		return nil, fmt.Errorf("Does not exist %s", CONFIG_PLIST+".plist")
+	plistPath := VMNetDomain + ".plist"
```
Comment (donmccasland) [inline pkg/drivers/hyperkit/network.go]: So it this not configurable anymore?

Comment (tstromberg) [reply pkg/drivers/hyperkit/network.go]: Still configurable - the variables were just renamed to fit the style guide: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Pull request merged**